lntest/harness_assertion.go

```diff
@@ -2514,6 +2514,56 @@ func (h *HarnessTest) AssertClosingTxInMempool(cp *lnrpc.ChannelPoint,
 	return closeTx
 }
 
+// AssertLocalFCInsightsForClosedChannels asserts that the passed chanPoint
+// is found in the list of passed closedChannels and also asserts that the
+// local force close insights is as expected.
+func (h *HarnessTest) AssertLocalFCInsightsForClosedChannels(
```

Comment:
Please make this describe what it is actually asserting. Maybe `AssertCloseSummaryContainsLocalFCInsights`